### PR TITLE
Added header to web page

### PR DESCRIPTION
**Pull request by bandjeo**

### CODE REVIEW

File: oss-cloud-frontend/src/main.js [hunk 1]
```diff
@@ -5,9 +5,11 @@ import VueRouter from "vue-router";
 
 import routes from "./routes";
 
+
```
Comment (banecogic) [inline oss-cloud-frontend/src/main.js]: Please dont insert unnecessary spacing. Usually one empty line is used.
There are multiple empty lines occurrences in `Header.vue` template section as well.
We can add https://eslint.org/docs/rules/no-multiple-empty-lines with max:1, but I'm not sure where it's configured when using vue-cli-service for linting.

Comment (banecogic) [inline oss-cloud-frontend/src/main.js]: Please dont insert unnecessary spacing. Usually one empty line is used.
There are multiple empty lines occurrences in Header.vue template section as well.
We can add https://eslint.org/docs/rules/no-multiple-empty-lines with max:1, but I'm not sure where it's configured when using vue-cli-service for linting.

Comment (bandjeo) [reply oss-cloud-frontend/src/main.js]: Extra lines removed.

File: oss-cloud-frontend/src/components/AddContributor.vue [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <template>
   <v-dialog v-model="dialog" max-width="600px">
     <template v-slot:activator="{ on }">
-      <v-btn color="primary" v-on="on">Add contributor</v-btn>
+      <v-btn color="#1533a1" class="mt-10 mr-10" v-on="on">Add contributor</v-btn>
```
Comment (SvetislavMarjanovic) [inline oss-cloud-frontend/src/components/AddContributor.vue]: avoid inluce css

**Pull request merged**